RELEASE-NOTES.txt

```diff
@@ -1,7 +1,11 @@
+17.8
+-----
+
+
 17.7
 -----
 * [***] Added blogging reminders. Choose which days you'd like to be reminded, and we'll send you a notification prompting you to post on your site
-* [**] (Don't apply to Jetpack app): Self hosted sites that do not use Jetpack can now manage (install, uninstall, activate, and deactivate) their plugins [#16675]
+* [** Does not apply to Jetpack app] Self hosted sites that do not use Jetpack can now manage (install, uninstall, activate, and deactivate) their plugins [#16675]
```

Comment:
I opened this in MacDown and it rendered as a link, so moved the "Jetpack notice" into the `[]` to avoid that 🤷‍♂️